presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,220 @@
+===================
+LDAP Authentication
+===================
+
+Presto can be configured to enable frontend LDAP authentication over
+HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
+drivers. At present only simple LDAP authentication mechanism involving
+user name and password is supported.
+
+To enable LDAP authentication for Presto, configuration changes are made on
+the Presto coordinator. No changes are required to the worker configuration;
+only the communication from the clients to the coordinator is authenticated.
+
+Presto Server Configuration
+---------------------------
+
+Environment Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _ldap_server:
+
+LDAP Server
+~~~~~~~~~~~
+
+The Presto client sends a user name and password to the coordinator and
+coordinator validates these credentials using an external LDAP service.
```

Comment:
Since this sentence doesn't describe configuration per say, perhaps move this above, after say, "At present only simple LDAP authentication mechanism involving user name and password is supported.".